packages/donor/src/components/basic/Menu/Menu.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  Menu as MuiMenu,
+  MenuProps as MuiMenuProps,
+  MenuItem as MuiMenuItem,
+} from "@mui/material";
+import { AnalyticsButtonType } from "@zm-blood-components/common";
+import { ReactElement } from "react";
+import { reportClick } from "../../../Analytics";
+
+export interface MenuProps extends MuiMenuProps {
```

Comment:
I'm not a fan of extending an external props object.
I prefer listing all the fields I need here, one by one.
This makes writing the stories much easier, and helps other developers understand what capabilities are used in this project.

Reply:
In this case, I agree.
Done